vue-frontend/src/components/jobs-thankyou/LatestBlog.vue

```diff
@@ -0,0 +1,247 @@
+                loading="lazy"
+                :alt="blog.title"
+              />
+            </div>
+            <div
+              class="tw-flex tw-flex-row tw-justify-between tw-my-[10px] tw-font-inter-regular tw-text-[0.875rem] tw-leading-[1.3125rem] tw-text-black-core/[0.6]"
+            >
+              <span class="tw-w-auto">
+                {{ blog.author }}
+              </span>
+            </div>
+            <div class="tw-my-[10px]">
+              <span
+                @click="openBlog(blog)"
+                class="tw-font-inter-semibold tw-text-[1.25rem] tw-leading-[1.875rem] tw-text-black-core[0.87] tw-cursor-pointer"
+              >
+                {{ blog.title }}</span
+              >
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="tw-hidden lg:tw-block">
+      <div class="tw-grid tw-grid-flow-col tw-gap-4 tw-mt-14">
+        <div class="tw-row-span-2 tw-col-span-3">
+          <div
+            v-if="blog1"
+            @mouseover="activeIndex = index"
+            @mouseleave="activeIndex = index"
+            @touchstart.passive="activeIndex = index"
+            @click.native="mixpanel.track('tap_blog_post')"
+            class="tw-w-full tw-border tw-border-black-core/[0.6] tw-rounded-[20px] tw-box-border tw-px-4 tw-pt-[1.3rem] tw-pb-[1rem] xl:tw-pb-[7.4rem] 2xl:tw-pb-[5.8rem]"
+          >
+            <div class="tw-flex tw-flex-col">
+              <div
+                @click="openBlog(blog1)"
+                class="tw-block tw-mt-[1rem] tw-w-full"
+              >
+                <img
+                  :src="blog1.thumbnail"
+                  class="tw-w-[640px] tw-h-[362px] tw-object-cover tw-rounded-lg tw-cursor-pointer"
+                  loading="lazy"
+                  :alt="blog1.title"
+                />
+              </div>
+              <div
+                class="tw-flex tw-flex-row tw-justify-between xl:tw-my-[20px] xl:tw-my-[10px] tw-font-inter-regular tw-text-[1.25rem] tw-leading-[1.578rem] tw-text-black-core/[0.6]"
+              >
+                <span>
+                  {{ blog1.author }}
+                </span>
+              </div>
+              <div class="tw-my-[10px]">
+                <span
+                  @click="openBlog(blog1)"
+                  class="tw-font-inter-semibold tw-text-[2rem] tw-leading-[3rem] tw-text-black-core[0.87] tw-cursor-pointer"
+                >
+                  {{ blog1.title }}</span
+                >
+              </div>
+              <div
+                class="tw-flex tw-flex-row tw-justify-between tw-my-[10px] tw-font-inter-regular tw-text-[1.625rem] tw-leading-[2.4375rem] tw-text-black-core/[0.6]"
+              >
+                <p class="tw-line-clamp-3">
+                  {{ blog1.description }}
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="tw-col-span-1 ...">
```

Comment:
What is meant by `...`?

Also, i don't think we need to take separate blogs for this UI. We can handle both (mobile and web UI) in single for loop using flex properties.

This is not correct way for this type of simple UI

Reply:
i need one blog in 1st column which is not possible using slice and 2 blogs in 2nd column in web UI. that's why i used this way.